ansible/k8s-k8sproxy.yml

```diff
@@ -0,0 +1,61 @@
+# Setup the K8s shared web proxy
+
+# Load hostvars
+- hosts: >
+    {{ idr_environment | default('k8s') }}-k8svae-hosts
+    {{ idr_environment | default('k8s') }}-k8strain-hosts
```

Comment:
k8strain is slotted for deprecation?

Reply:
Yes, but it was already written and deployed before we made that decision. In practice the second cluster has been extremely useful for prototyping the CI/CD gitlab work.